src/test/java/org/prebid/server/cache/CacheServiceTest.java

```diff
@@ -446,15 +447,19 @@ public void cacheBidsOpenrtbShouldPerformHttpRequestWithExpectedBody() throws IO
 
         // then
         // Second value is adm length for each
-        verify(metrics, times(1)).updateCacheCreativeSize(eq("accountId"), eq(0));
-        verify(metrics, times(2)).updateCacheCreativeSize(eq("accountId"), eq(4));
+        verify(metrics, times(1))
```

Comment:
Don't need to set `times(1)` since `org.mockito.Mockito#verify(T)` already does.